src/CMakeLists.txt

```diff
@@ -628,8 +625,13 @@ target_compile_definitions(
     "$<${_is_library}:HPX_PLUGIN_NAME_DEFAULT=hpx_$<TARGET_PROPERTY:NAME>>"
 )
 
-set(hpx_targets hpx hpx_no_wrap_main plugin component)
-set(hpx_internal_targets hpx_core hpx_interface hpx_interface_no_wrap_main)
+set(hpx_targets hpx wrap_main plugin component)
+set(hpx_internal_targets hpx_core hpx_interface hpx_interface_wrap_main)
+
+# Temporary for compatibility with projects like gridtools
+add_library(hpx_no_wrap_main INTERFACE)
+target_link_libraries(hpx_no_wrap_main INTERFACE hpx)
+list(APPEND hpx_targets hpx_no_wrap_main)
```

Comment:
I think we can remove this outright since the target didn't exist in 1.4.1.

Reply:
It was more for projects like DLA / gridtools which currently depend on that, just for them to make the changes and after can be removed without deprecation protocol

Reply:
Fair enough, good thinking. https://github.com/STEllAR-GROUP/hpx/issues/4811 is there as a reminder to remove it before 1.5.0.